router/core/transport.go

```diff
@@ -122,6 +123,15 @@ func (t TransportFactory) RoundTripper(transport http.RoundTripper, enableStream
 					}
 					span.SetAttributes(otel.WgOperationType.String(operation.opType))
 				}
+
+				reqContext := getRequestContext(r.Context())
```

Comment:
The operation context is inside the request context so the code above is not needed.

Reply:
Done